grype/Dockerfile

```diff
@@ -21,12 +21,16 @@ COPY LICENSE /licenses
 ### Use yum as root user
 USER root
 
+COPY docker-ce.repo /etc/yum.repos.d/docker-cd.repo
+
 ### Install packages
-RUN INSTALL_PKGS="coreutils python3-pip jq" && \
+RUN INSTALL_PKGS="coreutils python3-pip jq docker-ce docker-ce-cli containerd.io docker-compose-plugin" && \
```

Comment:
Are all of these packages necessary?

Reply:
the docker-compose-plugin can probably be dropped. The other three are required. containerd.io is the container runtime, docker-ce is the docker engine and docker-ce-cli is the cli for docker. Updating now.